### PR TITLE
Update the cert management documentation

### LINKED ISSUE #578: Update External issuer docs to point to 2.7 release
Once we cut the 2.7 release, we need to update the external issuer docs to link to it.

### PR DESCRIPTION
**Pull request by ihcsim**
Includes reference to stable-2.7. Added line breaks to introduce new paragraphs, and some rewording.

Fixes #578 

Signed-off-by: Ivan Sim <ivan@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/2/tasks/rotating_identity_certificates.md [hunk 1]
```diff
@@ -24,12 +24,17 @@ on how to do this.
 
 ## Prerequisites
 
-We are going to use [step 0.13.3](https://smallstep.com/cli/) and
-[jq 1.6](https://stedolan.github.io/jq/). We will also make use of the
-`linkerd check` CLI utility which will give us extra assurance that the state
-of our mTLS configuration is valid at any point. The minimum CLI version is
-[edge-19.12.3](https://github.com/linkerd/linkerd2/releases/tag/edge-19.12.3).
-To begin with, you can run:
+Upgrade your Linkerd CLI to [stable-2.7.0](https://github.com/linkerd/linkerd2/releases/tag/stable-2.7.0).
```
Comment (zaharidichev) [inline linkerd.io/content/2/tasks/rotating_identity_certificates.md]: Currently there is no stable released. So do we merge that after we release it or before?

Comment (ihcsim) [reply linkerd.io/content/2/tasks/rotating_identity_certificates.md]: I'm thinking we can merge this a few days after the RC is out. Meanwhile, reviews and approval will be great. Thanks.

File: linkerd.io/content/2/tasks/use_external_certs.md [hunk 2]
```diff
@@ -12,11 +12,15 @@ that are issued by the proxy. The trust root, certificates and private key
...
-In order to do that Linkerd [edge-19.10.5](https://github.com/linkerd/linkerd2/releases/tag/edge-19.10.5)
-introduces the functionality to read and continuously monitor Kubernetes
-secrets for certificates change and to auto-reload the `identity` component's
-in-memory issuer certificate. In the following lines, we will demonstrate how
-this allows integration with the external certificate management solution
+
+To ensure the certificate issuance process remains uninterrupted, Linkerd
+[stable-2.7.0](https://github.com/linkerd/linkerd2/releases/tag/stable-2.7.0)
+and [edge-19.10.5](https://github.com/linkerd/linkerd2/releases/tag/edge-19.10.5)
+introduced updates to the `identity` component, enabling it to seaminglessly
```
Comment (zaharidichev) [inline linkerd.io/content/2/tasks/use_external_certs.md]: seaminglessly -> seamlessly ?

File: linkerd.io/content/2/tasks/use_external_certs.md [hunk 3]
```diff
@@ -12,11 +12,15 @@ that are issued by the proxy. The trust root, certificates and private key
...
+auto-reload new mTLS issuer certificate.
```
Comment (zaharidichev) [inline linkerd.io/content/2/tasks/use_external_certs.md]: certificate -> certificates

**Comment by ihcsim** [comment]
The CI failure is caused by a ref to www.tcpdump.org, which is currently offline. I will retry CI after the site is back.

**Comment by ihcsim** [comment]
Superseded by https://github.com/linkerd/website/pull/655.

**Pull request closed without merging**